examples/inverse/plot_mixed_source_space_inverse.py

```diff
@@ -0,0 +1,135 @@
+"""
+=======================================================================
+Compute MNE inverse solution on evoked data in a mixed source space
+=======================================================================
+
+Create a mixed source space and compute MNE inverse solution on evoked dataset
```

Comment:
well add a note if MNE dSPM or sLORETA give the same result for this
analysis. ok?